Foxtrot/Tests/TestDriver.cs

```diff
@@ -130,7 +132,7 @@ private static void ExtractContracts(string assemblyFile, Options options)
                     ContractNodes contractNodes = null;
                     Extractor.ExtractContracts(
                         assemblyNode, null, null, null, null, out contractNodes,
-                        e => Assert.Fail(e.ToString()), false);
+                        e => Assert.True(false, e.ToString()), false);
```

Comment:
I'm using NUnit all the time, so I'm not super familiar with xUnit. Does this change means that there is no `Fail` method in xUnit? Because `Assert.Faile` is much more descriptive than `Assert.True(false)` IMO.